project/project/settings.py

```diff
@@ -23,9 +24,13 @@
 SECRET_KEY = '%5+eq2851!d7qi^sze(nv2g#kt8v$7)4ck3cq*e!5c2rx%13p+'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = bool(strtobool(os.getenv('DEBUG', 'False')))
 
 ALLOWED_HOSTS = []
+if DEBUG:
```

Comment:
Isn't that the default behaviour? https://docs.djangoproject.com/en/3.1/ref/settings/#allowed-hosts